src/main/resources/templates/java/gradle_gradle/exercise/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

Comment:
Maybe putting a readme document in `src/main/resources/templates/` next to the `error.html` would be fine without interfering with the actual templates.
Then the document could also be extended in the future by others with information about the other templates as needed. If it grows bigger, then it can always be moved to a proper documentation page in the `docs/dev/` directory (or you start a new page there directly).

Reply:
I created the README file in 230556a.